proxy_test.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"net"
```

Comment:
This should go up with the other stdlib imports. Also, `gofmt` and `goimports` should fix it for you.